recipes/cpr/all/test_package/conanfile.py

```diff
@@ -8,6 +8,8 @@ class TestPackageConan(ConanFile):
 
     def build(self):
         cmake = CMake(self)
+        cmake.definitions["ENABLE_SINGLE_PRECISION"] = self.options["fftw"].precision == "single"
+        cmake.definitions["ENABLE_LONG_DOUBLE_PRECISION"] = self.options["fftw"].precision == "longdouble"
```

Comment:
Is this needed? Are they related to `cpr` or is it legacy copy/paste from another package?

Reply:
Yes, copy/paste error!